frontend/config/application.rb

```diff
@@ -42,9 +42,6 @@ def self.extend_aspace_routes(routes_file)
     # https://github.com/rack/rack/issues/1075
     config.middleware.insert_before Rack::Runtime, MultipartBufferSize
 
-    # https://github.com/rack/rack/issues/1075
-    config.middleware.insert_before Rack::Runtime, MultipartBufferSize
```

Comment:
@cfitz I had to add this back in. It was causing an error (see below). It couldn't compute the holdings information and when I pulled up the locations browse page, I got the attached issue - spinners in the repo holdings column.
<img width="834" alt="screen shot 2017-10-17 at 9 05 38 am" src="https://user-images.githubusercontent.com/3585151/31667646-10d4a61e-b31e-11e7-89e4-ef939828d171.png">


I, [2017-10-17T08:51:34.205953 #1024]  INFO -- : Started GET "/search.json?filter_term=%7B%22location_uris%22%3A%22%2Flocations%2F384%22%7D&listing_only=true&type=top_container" for 0:0:0:0:0:0:0:1 at 2017-10-17 08:51:34 -0400
I, [2017-10-17T08:51:34.205970 #1024]  INFO -- : Started GET "/search.json?filter_term=%7B%22location_uris%22%3A%22%2Flocations%2F336%22%7D&listing_only=true&type=top_container" for 0:0:0:0:0:0:0:1 at 2017-10-17 08:51:34 -0400
F, [2017-10-17T08:51:34.225653 #1024] FATAL -- :   
F, [2017-10-17T08:51:34.226047 #1024] FATAL -- :   
F, [2017-10-17T08:51:34.228752 #1024] FATAL -- : NoMethodError (undefined method `+' for nil:NilClass):
F, [2017-10-17T08:51:34.228897 #1024] FATAL -- :   
F, [2017-10-17T08:51:34.229012 #1024] FATAL -- : NoMethodError (undefined method `+' for nil:NilClass):
F, [2017-10-17T08:51:34.229111 #1024] FATAL -- :   
F, [2017-10-17T08:51:34.229010 #1024] FATAL -- : config/application.rb:30:in `call'
F, [2017-10-17T08:51:34.229191 #1024] FATAL -- : config/application.rb:30:in `call'
F, [2017-10-17T08:51:34.229771 #1024] FATAL -- :   
F, [2017-10-17T08:51:34.231195 #1024] FATAL -- : NoMethodError (undefined method `+' for nil:NilClass):
F, [2017-10-17T08:51:34.231279 #1024] FATAL -- :   
F, [2017-10-17T08:51:34.231185 #1024] FATAL -- :   
F, [2017-10-17T08:51:34.231337 #1024] FATAL -- : config/application.rb:30:in `call'
F, [2017-10-17T08:51:34.233221 #1024] FATAL -- : NoMethodError (undefined method `+' for nil:NilClass):
F, [2017-10-17T08:51:34.233335 #1024] FATAL -- :   
F, [2017-10-17T08:51:34.233419 #1024] FATAL -- : config/application.rb:30:in `call'
F, [2017-10-17T08:51:34.233336 #1024] FATAL -- :   
F, [2017-10-17T08:51:34.235371 #1024] FATAL -- : NoMethodError (undefined method `+' for nil:NilClass):
F, [2017-10-17T08:51:34.235486 #1024] FATAL -- :   
F, [2017-10-17T08:51:34.235559 #1024] FATAL -- : config/application.rb:30:in `call'
F, [2017-10-17T08:51:34.235374 #1024] FATAL -- :   
F, [2017-10-17T08:51:34.237702 #1024] FATAL -- : NoMethodError (undefined method `+' for nil:NilClass):
F, [2017-10-17T08:51:34.237830 #1024] FATAL -- :   
F, [2017-10-17T08:51:34.237937 #1024] FATAL -- : config/application.rb:30:in `call'

Reply:
@cfitz I am going to try this as a fix - https://github.com/rhinobird-io/activity-plugin/commit/b4d7efbea89515d8e2b667501e61fcfbfc2c4411. Will let you know how that goes.

Reply:
@cfitz rhinobird-io/activity-plugin@b4d7efb worked. Can you do that in this PR? Then I think it is ready to go...